salt/annotations/init.sls

```diff
@@ -104,14 +104,3 @@ logrotate-for-annotations-logs:
     file.managed:
         - name: /etc/logrotate.d/annotations
         - source: salt://annotations/config/etc-logrotate.d-annotations
-
-{% if pillar.elife.env in ['dev', 'ci'] %}
```

Comment:
not needed anymore due to https://github.com/elifesciences/builder-base-formula/pull/107 that configures it.

Reply:
`dev` will mostly be needed only to reproduce a `prod` EC2 instance locally, while `ci` is for testing the formula itself. The project will only use containers directly for local development and `ci` testing.

Reply:
(until we try to move away from EC2 istances for hosting containers, which will see the whole formula vanish)

Reply:
> which would see the whole formula vanish

would it be pushed back into the cloudformation template?

Reply:
The option I see right now is to adopt Kubernetes (waiting for EKS availability), which will mean we will have some private repository (probably just a folder in `builder-private` in which to store configuration):
https://kubernetes.io/docs/concepts/configuration/overview/
Too soon to say where all the responsibilities end up, I suspect we can automate the interaction with Kubernetes within builder:
https://github.com/kubernetes-client/python
in order to allow a uniform tool to deploy both to ec2 and with containers:
https://queue.acm.org/detail.cfm?id=3158370